src/test/java/com/rocketden/main/UserTests.java

```diff
@@ -126,4 +130,35 @@ public void createNewUserNicknameContainsSpaces() throws Exception {
 
         assertEquals(ERROR.getResponse(), actual);
     }
+
+    @Test
+    public void deleteExistingUser() throws Exception {
+        CreateUserRequest createRequest = new CreateUserRequest();
+        createRequest.setNickname("rocket");
+        
+        DeleteUserRequest request = new DeleteUserRequest();
+        request.setNickname("rocket");
+
+        this.mockMvc.perform(post(POST_USER)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(Utility.convertObjectToJsonString(request)));
+
+        this.mockMvc.perform(delete(POST_USER)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(Utility.convertObjectToJsonString(request)))
+                .andExpect(status().isAccepted())
+                .andExpect(content().string("true"));
```

Comment:
I know `POST_USER` is the same value as `DELETE_USER`, but I think this makes it hard to follow from a readability standpoint. I'd recommend either renaming the variable or creating a duplicate variable named `DELETE_USER`.

Reply:
How does just `USER` sound?